docs/cloud/managing-airbyte-cloud/manage-airbyte-cloud-notifications.md

```diff
@@ -2,26 +2,35 @@
 
 This page provides guidance on how to manage notifications for Airbyte Cloud, allowing you to stay up-to-date on the activities in your workspace. 
 
-## Set up Webhook notifications
+
+## Configure Notification Settings
 
 To set up Webhook notifications:
 
 1. On the [Airbyte Cloud](http://cloud.airbyte.com) dashboard, click **Settings**.
 
 2. Click **Notifications**.
 
-3. [Create an Incoming Webhook for Slack](https://api.slack.com/messaging/webhooks).
+3. Have a Slack Webhook ready if you plan to use Slack Webhook notification. [Create an Incoming Webhook for Slack](https://api.slack.com/messaging/webhooks).
+
+4. Toggle type of events you are interested to receive notifications. 
+	1. If using slack webhook, please fill in your webhook address. For your convenience, we provide a 'test' function to send a test message to your slack channel so you can make sure everything has been connected.
+
+5. Click **Save changes**.
 
-4. Navigate back to the Airbyte Cloud dashboard > Settings > Notifications and enter the Webhook URL.
+## Notification Event Types
 
-5. Toggle the **When sync fails** and **When sync succeeds** buttons as required.
+1. Failed syncs and Successful syncs: When a connection sync has finished, you can choose to be notified with if the sync has failed, successful or both. Note if sync runs frequently or if there are many syncs in the workspace these types of events can be noisy.
+1. Automated Connection Updates: when a connection detects source schema has changed and can be updated automatically, Airbyte will update you connection and send you a notification message.
+1. Connection Updates Requiring Action: When a connection detects some updates that requires action to run syncs. Since this will affect your sync from running as scheduled, you cannot disable the email notification for this type of event.
+1. Sync Disabled and Sync disabled Warning: If a sync has been failing for multiple days or many times consecutively, we will disable the connection to prevent it run further. We will send a sync disabled warning notification when we detects the trend, and once the failure counts hits our threshold we will send a sync disabled notification and will actually disable the connection. Again, because the sync will not continue to run as you have configured, we made the decision that you cannot disable the email notification for 'Sync Disabled' event so you will always get notified.
```

Comment:
```suggestion
1. Sync Disabled and Sync Disabled Warning: If a sync has been failing for multiple days or many times consecutively, Airbyte will disable the connection to prevent it from running further. Airbyte will send a Sync Disabled Warning notification when we detect the trend, and once the failure counts hits a threshold Airbyte will send a Sync Disabled notification and will actually disable the connection. Again, because the sync will not continue to run as you have configured, the Sync Disabled notification cannot be disabled.
```